apps/api/src/common/EnvVars.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Environments variables declared here.
+ */
+
+/* eslint-disable node/no-process-env */
+
+
+export default {
+  NodeEnv: (process.env.NODE_ENV ?? ''),
+  Port: (process.env.PORT ?? 0),
+  CookieProps: {
+    Key: 'ExpressGeneratorTs',
+    Secret: (process.env.COOKIE_SECRET ?? ''),
+    // Casing to match express cookie options
+    Options: {
+      httpOnly: true,
+      signed: true,
+      path: (process.env.COOKIE_PATH ?? ''),
+      maxAge: Number(process.env.COOKIE_EXP ?? 0),
+      domain: (process.env.COOKIE_DOMAIN ?? ''),
+      secure: (process.env.SECURE_COOKIE === 'true'),
+    },
+  },
+  Jwt: {
+    Secret: (process.env.JWT_SECRET ??  ''),
+    Exp: (process.env.COOKIE_EXP ?? ''), // exp at the same time as the cookie
+  },
+} as const;
```

Comment:
Use the same package t3-env no?

Reply:
looking at the benefits I don't think it justifies another dependency, went for simple types with zod